extensions/nginx/acme.js

```diff
@@ -27,10 +27,6 @@ function install(ui, task) {
         ui.logVerbose('ssl: downloading acme.sh to temporary directory', 'green');
         return fs.emptyDir(acmeTmpDir)
     }).then(() => got(acmeApiUrl)).then((response) => {
-        if (response.statusCode !== 200) {
```

Comment:
I removed this part from here, as `got` should never resolve if the `statusCode` is not `2xx`. See: https://github.com/sindresorhus/got#gothttperror

Unless you can think of any other occurrence?

Reply:
I'm pretty sure that was me being a goof 😜